bin/lib/Logos/Generator/Base/Property.pm

```diff
@@ -5,19 +5,33 @@ use Logos::Util;
 sub getterName {
 	my $self = shift;
 	my $property = shift;
-	return Logos::sigil("method", $property->group->name, $property->class->name, $property->getter);
+	return Logos::sigil("method", $property->group->name, $property->class->name, $property->getter, "getter");
 }
 
 sub setterName {
 	my $self = shift;
 	my $property = shift;
-	return Logos::sigil("method", $property->group->name, $property->class->name, $property->setter);
+	return Logos::sigil("method", $property->group->name, $property->class->name, $property->getter, "setter");
+}
+
+# addAttribute(attributeList, attributeCount, attributeName, attributeValue)
+#
+# attributeName is one of the property attribute characters, it will be "stringified" for you
+# attribute value must be a variable or something else; it will NOT be "stringified" for you; omit it to use ""
+sub addAttribute {
+	my $self = shift;
+	my $attrList = shift;
+	my $counter = shift;
+	my $name = shift;
+	my $value = shift // "\"\"";
+	return " $attrList [$counter++] = (objc_property_attribute_t) { \"$name\", $value };";
```

Comment:
Remove the space in `$attrList [$counter++]`

Reply:
Without the space it thinks `attrList` is a global variable. Can you clone this and see if you can figure out how to work around it?

Reply:
Do you mean that Perl thinks its a variable?
You can do this then:
`return " ".$attrList."[$counter++] = ...`

Reply:
Yeah. Thanks!